recipes/lazrs-python/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "lazrs-python" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/lazrs/lazrs-{{ version }}.tar.gz
+  sha256: 1ddb597bf5f42eb2977a8eb8ae08d0b3c8106cace3665fc7e6aea8adda0b8e60
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+  host:
+    - maturin
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - lazrs
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/lazrs/
+  summary: Python bindings for laz-rs
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - weiji14
```

@tmontaigu, would you like to be a conda-forge recipe maintainer for `package-name` (need a yes/no answer)? Typically we'll just need to merge in Pull Requests generated automatically by [conda-forge-bot](https://github.com/conda-forge-bot) after a PyPI release. Perhaps update the dependency list if it has changed, but otherwise, this shouldn't take too much time usually.

```suggestion
  recipe-maintainers:
    - tmontaigu
    - weiji14
```

---

yes i think that is ok